# 16S rRNA Gene Analysis

A highlight of many Tiny Earth courses is performing 16S rRNA Gene Analyses to identify the Genus of one's Isolates. To create an entry of this type, visit [My Entries](https://discovery.tinyearth.wisc.edu/my-entries/), locate the Isolate that was analyzed, and click "Add rrna-analysis" under that Isolate.

## Use This Entry to Record

- The Genus of an Isolate
- The method by which that was determined

## Required Information

- **Date:** The date the 16S rRNA was first tested. 
- **Isolate:** This field is automatically populated for you if you click "Add rrna-analysis" under an existing Isolate. If it is not automatically populated, you can search through the dropdown box and select the Isolate that you are working on. If a teammate created the Isolate entry, make sure that they have added you as a co-author so you may refer to it in your entries. Click the "add new" button only if you need to create a new Isolate from scratch to link to this entry.
- **Genus:** You can get the name of the genus from NCBI Blast. A drop-down list will appear when you click on the box. You can either scroll through the list or search for the genus. Be sure to confirm with your instructor that you are adding the correct genus.

## Optional Information

- **Nickname:** Please follow your instructor's guidelines on how to name entries meaningfully.
- **PCR Primers Used:** A list of primers is provided along with an option at the bottom to add more. Note that when you add a primer, it will not show up as an option for future entries. You may select one or multiple primers for every entry you log.
- **Polymerase Used:** You can also get this information from NCBI Blast. For this entry, you will only be able to select one polymerase. You can select an option provided or add your own by selecting the white box at the bottom of the list then writing in what polymerase was used.
- **Percent Identity:** This is the percent of the isolate's sequence that matches the reference organism's sequence. You can find this with your results in NCBI Blast. You can type your answer into the box or use the arrows to the right of the box to answer the prompt. 
- **FASTA:** A FASTA is formatted so that there is a greater than sign followed by a single-line description, then the sequence data on the following lines. You can copy your text into the entry box provided. An example of FASTA text is provided below.
```fasta
>P01013 GENE X PROTEIN (OVALBUMIN-RELATED)
QIKDLLVSSSTDLDTTLVLVNAIYFKGMWKTAFNAEDTREMPFHVTKQESKPVQMMCMNNSFNVATLPAE
KMKILELPFASGDLSMLVLLPDEVSDLERIEKTINFEKLTEWTNPNTMEKRRVKVYLPQMKIEEKYNLTS
VLMALGMTDLFIPSANLTGISSAESLKISQAVHGAFMELSEDGIEMAGSTGVIEDIKHSPESEQFRADHP
FLFLIKHNPTNTIVYFGRYWSP
```
- **Query Cover** and **Length:** You will also be able to gather this information from NCBI Blast. The query cover and length indicate the size of each subject sequence compared to the reference sequence. Query cover is a number that should represent what percent of the DNA reference sequence is being covered by the subject sequence. If you are unsure how to interpret your results, see your instructor for guidance. 

## Suggestions for "Additional Notes"

- `TODO`
